src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -215,5 +218,35 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
             Assert.Equal(0, target.ExitCode);
             Assert.Equal(0, result);
         }
+
+        [Fact]
+        public void StrykerCLI_WithNoFilesToExcludeSet_ShouldPassDefaultValueToStryker()
+        {
+            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
+            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>()));
+
+            var target = new StrykerCLI(mock.Object);
+
+            target.Run(new string[] { });
+
+            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
+                !o.FilesToExclude.Any())));
+        }
+
+        [Theory]
+        [InlineData("--files-to-exclude")]
+        [InlineData("-fte")]
+        public void StrykerCLI_WithFilesToExcludeSet_ShouldPassFilesToExcludeToStryker(string argName)
+        {
+            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
+            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>()));
+
+            var target = new StrykerCLI(mock.Object);
+
+            target.Run(new[] { argName, "['/StartUp.cs']" });
```

Comment:
Agreed, Only I was thinking, now I have a JSON format that you can apply e.g. '['C:\\ExampleProject\\StartUp.cs','C:\\ExampleProject\\Recursive.cs']' but that looks ugly. Maybe it is a better idea to do something like this 'C:\\ExampleProject\\StartUp.cs,C:\\ExampleProject\\Recursive.cs'? It is still something what is bugging me and I can't think about a better idea how to provide it.

Reply:
I think `['C:\ExampleProject\StartUp.cs','C:\ExampleProject\Recursive.cs']` is acceptable so long as it can be one line. For bigger projects with more excludes the file config should be used anyway.

Reply:
True, I'll update everything to JSON input. In the new issue I'll try to think about of how to resolve it in a more easy and fancy way :-).

Reply:
I have to admit that providing a JSON string through a unit test is quite a challenge. Somehow the Newtonsoft Serializer has really big trouble translating the input. I have it solved but it needs some rework with fetching data which you have already stated in the issue #203 and it looks really yuck ... I've committed the changes so you can review them.